pkgs/tools/security/metasploit/default.nix

```diff
@@ -1,13 +1,13 @@
-{ stdenv, fetchgit, makeWrapper, ruby, bundler, bundlerEnv, git, postgresql, libpcap, sqlite, zlib }:
+{ stdenv, fetchurl, makeWrapper, ruby, bundler, bundlerEnv, git, postgresql, libpcap, sqlite, zlib }:
 
 stdenv.mkDerivation rec {
   name = "metasploit-framework-${version}";
   version = "4.12.5";
 
 
-  src = fetchgit {
-    url = "https://github.com/rapid7/metasploit-framework.git";
-    sha256 = "0rzn2lax3nvhgmfivr7jdifir9j8f1f7mk2ypksr8lvqk4hr54kw";
+  src = fetchurl {
+    url = "https://github.com/rapid7/metasploit-framework/archive/4.12.5.tar.gz";
```

Comment:
it's better to use `fetchFromGitHub { owner = "rapid7"; repo = "metasploit-framework"; rev = "4.12.5"; sha256 = "..."; }`